code/modules/power/portgen.dm

```diff
@@ -84,21 +84,24 @@
 		spawn(duration)
 			stat &= ~EMPED
 
-/obj/machinery/power/port_gen/proc/explode()
+/obj/machinery/power/portgen/proc/explode()
 	explosion(loc, -1, 3, 5, -1)
 	qdel(src)
 
 #define TEMPERATURE_DIVISOR 40
 #define TEMPERATURE_CHANGE_MAX 20
 
-//A power generator that runs on solid plasma sheets.
-/obj/machinery/power/port_gen/pacman
-	name = "\improper P.A.C.M.A.N.-type Portable Generator"
-	desc = "A power generator that runs on solid phoron sheets. Rated for 80 kW max safe output."
+//
+// Portable Generator - Basic
+// Runs on phoron sheets.
+//
+/obj/machinery/power/portgen/basic
+	name = "portable generator"
+	desc = "A portable generator that runs on " + SPAN_INFO("solid phoron sheets. ") + SPAN_WARNING("Rated for 80 kW max safe output.")
 
 	var/sheet_name = "Phoron Sheets"
 	var/sheet_path = /obj/item/stack/material/phoron
```

Comment:
maybe i should change this to some other fuel, like coal, since phoron is supposed to be rare. otherwise i doubt that we're going to see the basic generator get used.